### DISCUSSION TITLE
Homebrew on Linux doesn't show option to install inside local user.

### DISCUSSION THREAD

**Discussion by tazihad** [Homebrew/discussions#3932] [Linux]
I am using Fedora Silverblue. There used to be an option to install homebrew on ~/.linuxbrew without root. But I don't see it anymore all it prompt me is use password to install brew in /home/homebrew. Any solution?
![Screenshot from 2022-11-17 11-03-03](https://user-images.githubusercontent.com/19417232/202360759-e142e068-e435-4428-bbe8-6c10d1ba1fa5.png)

![Screenshot from 2022-11-17 11-06-00](https://user-images.githubusercontent.com/19417232/202360942-afc2cf25-85b5-492f-b334-98f04c6a95eb.png)

**Comment by SMillerDev** [comment]
https://docs.brew.sh/Installation has some options
